src/test/java/stepDefinitions/CommonSteps.java

```diff
@@ -14,13 +15,20 @@ public class CommonSteps {
     pages.commonMethods commonMethods = new commonMethods();
 
     @Given("click on submit button")
-    public void click_on_submit_button() throws InterruptedException {
-        commonPage.clickOnContinueBtn();
+    public void i_click_on_submit_button() throws InterruptedException {
```

Comment:
Is it part of the convention to have _ in the method name iClickOnSubmitButton would this not work